department-of-veterans-affairs/caseflow
# Display missing veteran email alert

Resolves #15056

### Description
- add logic to `AppellantSection` to display  info `Alert` component when Veteran/Appellant email is missing
- create new story and write jest
- refactor and create some data in `Appeal.js`

### Testing Plan
1. run `make storybook`
2. go to storybook and see `HearingConversionTypeForm` 
**[Note about padding issue in Storybook](https://dsva.slack.com/archives/C3EAF3Q15/p1600116202201700?thread_ts=1600092053.191800&cid=C3EAF3Q15)**
### Storybook Story
*For Frontend (Presentational) Components*
